packages/ui/src/proposals/constants/proposals.ts

```diff
@@ -0,0 +1,5 @@
+import { ProposalStage } from '../types'
+
+export const proposalStages: ProposalStage[] = ['DECIDING', 'DORMANT', 'GRACING', 'SUCCEEDED']
+
+export const proposalTypes = ['FUNDING REQUEST']
```

Comment:
This should be as `ProposalType` type I think. You can also use it for the `Proposal.type` instead of `string`.

Reply:
But if you use those in the mocks only, please move them to the mocks so they would not interfere with the code. If so the constants file can be moved to `mocks` without the changes.

Reply:
Thanks for the review.

You're right these constants are only used by the `mockProposals` code so I just moved them there.